scripts/release/update-version.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+set -u
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+if [ $# -eq 0 ]; then
+    echo "Usage: v<semantic version>"
+    exit 1
+fi
+
+SEM_VERSION=$1
+SEM_VERSION=${SEM_VERSION/#v}
+SHORT_VERSION=${SEM_VERSION%-*}
+
+step "Version ${SEM_VERSION}"
+
+step "Updating CocoaPods podspecs to version ${SEM_VERSION}…"
+
+find . -type f -name '*.podspec' -exec sed -i '' "s/^ *version *=.*$/  version = '${SEM_VERSION}'/" {} +
```

Comment:
why are we updating the podspec here?

Reply:
This is a port from <10.0. This is likely to clash with your PR 😄